lms/static/js/bookmarks/views/bookmark_button.js

```diff
@@ -1,6 +1,6 @@
 ;(function (define, undefined) {
     'use strict';
-    define(['gettext', 'jquery', 'underscore', 'backbone', 'js/views/message_banner'],
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'lmsCoursewareMessageBanner'],
         function (gettext, $, _, Backbone, MessageBannerView) {
```

Comment:
I have already tried that but for whatever reason its not working, anyway i will give it another try.